static/components/customer-stall/customer-stall.html

```diff
@@ -141,7 +141,8 @@
           filled
           dense
           v-model.trim="checkoutDialog.data.address"
-          label="Address"
+          label="Address *optional"
```

Comment:
`Address *optional` -> `Address (optional)`

Reply:
it's for consistency with the rest! I'll change them all